src/pages/Registration.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { Link, Redirect } from "react-router-dom";
+import axios from 'axios';
+import logoImg from "../img/logo.jpg";
+import { Card, Logo, Form, Input, Button, Error } from "../components/AuthForms";
+import { useAuth } from "../context/auth";
+
+function Registration(props) {
+    const [isLoggedIn, setLoggedIn] = useState(false);
+    const [isError, setIsError] = useState(false);
+    const [userName, setUserName] = useState("");
+    const [password, setPassword] = useState("");
+    const [checkPassword, setCheckPassword] = useState("");
+    const [email, setEmail] = useState("");
+    const { setAuthTokens } = useAuth();
+    const referer =  '/';
+
+    function postRegistration() {
+            if (password !== checkPassword) {
+                alert("Passwords don't match");
+            } else {
+                axios.post("auth/registration", {
```

Comment:
Did you really created a new user with this path??? I checked the swagger and found there url `/auth/register`

Reply:
No, I have not tested with the server